requirements.txt

```diff
@@ -81,7 +81,7 @@ passlib==1.7.1
 pluggy==0.13.0
 promise==2.2.1
 protobuf==3.10.0
-psycopg2==2.8.3
+psycopg2-binary==2.8.3
 py==1.8.0
```

Comment:
@arielszabo  why use the binary version? and why do you need to fix the version? (we just removed the fixed versions from the requirements.txt file)

Reply:
@atalyaalon I just didn't want to count on these packages to be backwards compatible so I fixed the version numbers  - your call if you want to change it. :)
The non-binary version just caused me some errors while building the docker.

Reply:
for now, don't fix versions. regarding the binary package - that's fine.

Reply:
@atalyaalon Done  :)